.gitignore

```
@@ -2,3 +2,5 @@ _cache
 bin
 *.patch
 *sync.sh
+build.sh
+bundle
```

We can remove `bundle.Dockerfile` as well 

```suggestion
bundle
bundle.Dockerfile
```